pkg/azuredisk/azuredisk_v2.go

```diff
@@ -89,6 +92,25 @@ const (
 	azDriverNodeHealthy      azDriverNodeStatus = "Driver node healthy."
 )
 
+// LatencyAdapter implements LatencyMetric.
+type LatencyAdapter struct {
```

Comment:
These changes are part of https://github.com/kubernetes-sigs/azuredisk-csi-driver/pull/1410 so you should probably leave these changes our in this PR.